blueprints/using-the-fly-docker-registry.html.md

```diff
@@ -0,0 +1,118 @@
+---
+title: Managing Docker Images with Fly.io's Private Registry
```

Comment:
I might make the file on disk match the title, so then naming the file managing-docker-images-with-flyios-private-registry.html.md.